rust/src/fbs.rs

```diff
@@ -1,9 +1,12 @@
-pub use root::*;
+//! Flatbuffers data structures compiled from `.fbs` files
+pub use root::fbs::*;
 
 const _: () = ::planus::check_version_compatibility("planus-0.3.1");
 
 #[no_implicit_prelude]
 mod root {
+    #[allow(missing_docs)]
+    #[allow(clippy::all)]
```

Comment:
This file is auto-generated. Do we need to add this changes manually when the flatbuffers schema changes?

Reply:
I believe the way you had it before could suppress the whole project. This will is currently committed into the repo, but it will be replaced with actually auto-generated during compile time in `mediasoup-sys` in the future, so these changes are temporary either way.